cmd/devtools/run.go

```diff
@@ -57,6 +57,19 @@ func runCmdHandler(c *cobra.Command, args []string) {
 	instance := c.Flag("instance").Value.String()
 	IsInstanceNameValidOrPanic(instance)
 
+	// create log file for the UI app
```

Comment:
We've got the `cmd/logging.go` file. Can this logic go in a public function that lives there?